chia/rpc/wallet_rpc_api.py

```diff
@@ -1387,11 +1387,14 @@ async def dl_update_root(self, request) -> Dict:
 
         for _, wallet in self.service.wallet_state_manager.wallets.items():
```

Comment:
How about using `.values()` and eliminate the `_`

Reply:
Yes I keep forgetting I can do this :)